build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.12.0-1
+v1.13.0-0
```

Comment:
the version is actually based on the underlying golang version, so this should be `v1.12.0-1`.

Reply:
Ack. switching it back